src/fw/freertos_application.c

```diff
@@ -89,12 +85,7 @@ extern void vPortSuppressTicksAndSleep( TickType_t xExpectedIdleTime ) {
   // interrupts with priorities lower than configMAX_SYSCALL_INTERRUPT_PRIORITY
   // from executing and from waking the processor.
   // See: http://infocenter.arm.com/help/topic/com.arm.doc.dui0552a/BABGGICD.html#BGBHDHAI
-#if MICRO_FAMILY_NRF52840
-  uint8_t is_nest;
-  sd_nvic_critical_region_enter(&is_nest);
-#else
   __disable_irq();
-#endif
```

Comment:
though this brings up the interesting question as to whether NimBLE is going to need an analogous change later